src/main/java/com/lazerycode/jmeter/mojo/AbstractJMeterMojo.java

```diff
@@ -67,6 +67,13 @@
 	protected boolean appendResultsTimestamp;
 
 	/**
+	 * Set the format of the results generated by JMeter
+	 * Valid values are: xml, csv (XML set by default).
+	 */
+	@Parameter(defaultValue = "xml")
+	protected String resultsFileFormat;
+
```

Comment:
this is in the `AbstractJMeterMojo`. `AbstractMojo` is a class from the Maven API.